lib/widgets/app.dart

```diff
@@ -58,24 +59,67 @@ class ChooseAccountPage extends StatelessWidget {
     final globalStore = GlobalStoreWidget.of(context);
     return Scaffold(
       appBar: AppBar(title: const Text('Choose account')),
-      body: Center(
-        child: ConstrainedBox(
-          constraints: const BoxConstraints(maxWidth: 400),
-          child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
-            for (final (:accountId, :account) in globalStore.accountEntries)
-              _buildAccountItem(context,
-                accountId: accountId,
-                title: Text(account.realmUrl.toString()),
-                subtitle: Text(account.email)),
-            const SizedBox(height: 12),
-            ElevatedButton(
+      body: Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: [
+        Builder(builder: (context) =>
+          MediaQuery.removePadding(
+            context: context,
+            removeBottom: true,
+            child: Expanded(
+              child: SingleChildScrollView(
```

Comment:
This gets pretty long before it becomes clear what the payload is (with `for (… in globalStore.accountEntries)` and `_buildAccountItem`). It'd be good to pull out as a local variable.